app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -27,14 +27,14 @@ import org.mozilla.focus.tiles.HomeTile
 import org.mozilla.focus.tiles.CustomTilesAccessor
 
 private const val COL_COUNT = 5
+const val HOME_TILES_DIR = "defaults/"
```

Comment:
nit: private? Can we move the other constants out too? (this file is really hard to read)